.github/workflows/test-metal3-helm-chart.yml

```diff
@@ -0,0 +1,127 @@
+name: Test Metal3 Helm Chart
+
+on:
+  pull_request:
+    paths:
+      - '**/assets/metal3/**'  # Adjust the path based on your Helm chart structure
+
+jobs:
+  test-metal3-helm-chart:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout Repository
+      uses: actions/checkout@v3
+
+    - name: Get changed files
+      id: changed-files
+      uses: tj-actions/changed-files@v39
+        
+    - name: Verify and copy metal3 assets 
+      run: |
+        metal3_archive=$(echo "${{ steps.changed-files.outputs.all_changed_files }}" | tr -s " " "\012" | grep assets/metal3/)
+        
+        # Count the number of files in the array
+        num_files=$(echo "${metal3_archive}" | wc -l)
+        # Check if there is only one file
+        if [ "${num_files}" -gt 1 ]; then
+          echo "Multiple files found in assets/metal3:"
```

Comment:
Perhaps we can make this more explicit? - "Multiple archives modified - please modify only a single chart release per PR" or something?